docs/apache-airflow/howto/docker-compose/docker-compose.yaml

```diff
@@ -74,6 +75,10 @@ x-airflow-common:
     - ${AIRFLOW_PROJ_DIR:-.}/logs:/opt/airflow/logs
     - ${AIRFLOW_PROJ_DIR:-.}/config:/opt/airflow/config
     - ${AIRFLOW_PROJ_DIR:-.}/plugins:/opt/airflow/plugins
+    #The following line can be used to use a custom airflow.cfg, stored in the AIRFLOW_PROJ_DIR
+    #Uncomment the following line to make use of this
+    #- ${AIRFLOW_PROJ_DIR:-.}/airflow.cfg:/opt/airflow/airflow.cfg
+
```

Comment:
Another option place config, e.g. `awesome.cfg` into the `config` directory and setup [`AIRFLOW_CONFIG`](https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html#envvar-AIRFLOW_CONFIG) environment variable to `/opt/airflow/config/awesome.cfg`

Reply:
Yep. Lokely better